sonic-net/sonic-buildimage
# platform/marvell renaming to platform/marvell-prestera

#### Why I did it
Renaming of platforms and asic type for marvell
This PR has dependencies on the below submodule PRs
<html><body>

Repo | PR title | State
-- | -- | --
sonic-swss |[ platform/marvell renaming to platform/marvell-prestera](https://github.com/sonic-net/sonic-swss/pull/3378) | Merged
sonic-utilities |[ platform/marvell renaming to platform/marvell-prestera](https://github.com/sonic-net/sonic-utilities/pull/3622) | Merged
sonic-sairedis |[ platform/marvell renaming to platform/marvell-prestera](https://github.com/sonic-net/sonic-sairedis/pull/1465) | Merged
sonic-linux-kernel |[Rename marvell to marvell-prestera](https://github.com/sonic-net/sonic-linux-kernel/pull/457) | Merged
sonic-mgmt |[Changes to support asic name change from marvell to marvell-prestera](https://github.com/sonic-net/sonic-mgmt/pull/17171) | Merged

</body>
</html>

##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it
renamed platform folder names and asic types as below
platform/marvell renamed to platform/marvell-prestera

#### How to verify it
Verify marvell-prestera builds going through, and verify the images in prestera switches and verified linkup of interfaces
verified below architecture of switches and build for marvell-prestera
arm64
armhf
amd64



#### Which release branch to backport (provide reason below if selected)




#### Tested branch (Please provide the tested image version)




#### Description for the changelog




#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
